src/sidebar/styles.css

```diff
@@ -1,23 +1,23 @@
 html {
-	height: 100%;
+  height: 100%;
 }
 
 body {
-	height: 100%;
-	margin: 0;
-	overflow: hidden;
-	padding: 0;
+  height: 100%;
+  margin: 0;
+  overflow: hidden;
+  padding: 0;
 }
 
 div#editor {
-	height: 90%;
-	height: calc(100% - 70px);
+  height: 90%;
+  height: calc(100% - 70px);
```

Comment:
Just noticed that we define `height` twice here... I'll do some CSS linting and file a separate bug.